workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/movies.py

```diff
@@ -202,6 +202,14 @@ def register_parser(subparsers):
         type=int,
         help="Number of timesteps to use in movie. If not provided all times are used.",
     )
+    parser.add_argument(
+        "--from_start",
+        action="store_true",
+        help=(
+            "If true, generate the movie of n_timesteps at the start of the run. "
+            "If false, use last n_timesteps of the run."
+        ),
```

Comment:
```suggestion
        "--from_end",
        action="store_true",
        help="Use end of run for movie instead of start of run.",
```
can we have the default behavior remain unchanged? i.e. can we have the flag be called `from_end` and if it is not provided, then the movie is made from the start of the run.

Reply:
Whoops, forgot the default is the opposite of the action. Fixed.